README

```diff
@@ -151,6 +151,7 @@ REQUIREMENTS:
         libcryptsetup (optional), >= 2.3.0 required for signed Verity images support
         libaudit (optional)
         libacl (optional)
+        libbpf >= 0.2.0 (optional)
```

Comment:
Thanks for adding this - would it be possible to also document the kernel side requirements? There's a section above with required and optional kconfigs for example. Thanks!

Reply:
@bluca updated: 49a734a

Reply:
Thanks!